src/Plugin/MediaEntity/Type/Twitter.php

```diff
@@ -163,6 +176,20 @@ public function getField(MediaInterface $media, $name) {
             return $tweet['retweet_count'];
           }
           return FALSE;
+
+        case 'thumbnail':
+          $uri = 'public://twitter-thumbnails/' . $matches['id'] . '.svg';
+          if (! file_exists($uri)) {
+            $build = [
+              '#theme' => 'media_entity_twitter_tweet_thumbnail',
+              '#tweet' => $tweet['text'],
+              '#author' => $tweet['user']['name'],
+              '#avatar' => $tweet['user']['profile_image_url'],
```

Comment:
What happens if user changes profile image. Our SVG will still point to the old one, which will be gone at that point.